code/.storybook/manager.tsx

```diff
@@ -1,9 +1,25 @@
-import { addons } from 'storybook/internal/manager-api';
+import { type State, addons } from 'storybook/internal/manager-api';
```

Comment:
This code is temporary (to simplify showcase if someone wants to check out the branch) and will be removed prior to merging.